lib/membrane_element_mpeg_audio/depayloader.ex

```diff
@@ -15,17 +15,13 @@ defmodule Membrane.RTP.MPEGAudio.Depayloader do
   Frag_offset: Byte offset into the audio frame for the data in this packet.
   """
   use Membrane.Filter
-  alias Membrane.Buffer
-  alias Membrane.RTP
+  alias Membrane.{Buffer, RTP, RemoteStream}
+  alias Membrane.Caps.Audio.MPEG
 
   @default_demand 1
 
-  def_output_pad :output,
-    caps: :any
-
-  def_input_pad :input,
-    caps: {RTP, payload_type: 14},
-    demand_unit: :buffers
+  def_input_pad :input, caps: RTP, demand_unit: :buffers
+  def_output_pad :output, caps: {RemoteStream, content_format: MPEG, type: :packetized}
```

Comment:
Why do we remove `payload_type: 14`?

Also we should change `handle_caps` so that it returns sth like
```elixir
    caps = %RemoteStream{content_format: MPEG, type: :packetized}
    {{:ok, caps: {:output, caps}}, state}
```

Reply:
`payload_type` field was removed from RTP format because the RTP payload is usually independent of the payload type. Also, there can be multiple streams with different payload types multiplexed.